package.json

```diff
@@ -17,13 +17,8 @@
     "url": "https://github.com/0xfe/vexflow/issues"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-transform-object-assign": "^7.8.3",
-    "@babel/preset-env": "^7.8.4",
```

Comment:
This might be a question from ignorance, but Isn't it premature to remove babel while there is still so much source in JS?  Does Typescript on JS sources really do all of the transformations that are needed?  If so, then it's a step I can remove in many of my sources.  But if we're still building to ES5 we will probably need the core.js features.

Reply:
ts_loader is now compiling/transforming all the files
it is configured with javascript support and ES5 as target
https://github.com/0xfe/vexflow/blob/1924b2cc268de33fb19bbc8ba62c59b635614ea4/tsconfig.json#L1-L5

Reply:
This is a bit scary to me too, but a good change if it does what it's supposed to do. Are you running the full visual regression tests?

Reply:
@0xfe  how do you that? I am doing grunt test & npm start.

Reply:
`npm test` will do it. See https://github.com/0xfe/vexflow/wiki/Visual-Regression-Tests.

Reply:
Done, I get: **Success - All diffs under threshold!**

Reply:
Oh, I understand why I was confused -- I've never used typescript alone to target a specific ECMAScript target but generally used TypeScript with babel/browserlist/core-js to ensure that I'm outputting to a level of compliance that allows using features like WeakMap and native classes (for custom elements); if we're going to pure ES5 then there's no need.